### ISSUE TITLE
503: name 'tonlib' is not defined

### ISSUE THREAD

**Issue by ahmeteminkocal** [toncenter/ton-http-api#105]
{ ok: false, error: "name 'tonlib' is not defined", code: 503 }
I started getting this error on withdrawals. What could be the issue?

**Comment by tiennguyen12g** [comment]
Did you fix this issue? Today I have the same error. Please share me the solution. Thank bro

**Comment by ahmeteminkocal** [comment]
> Did you fix this issue? Today I have the same error. Please share me the solution. Thank bro

yes don't use default network node which is *something*network (I forgot its name).
Define node URL manually, either by installing a node yourself or find a public node.

**Comment by tiennguyen12g** [comment]
> > Did you fix this issue? Today I have the same error. Please share me the solution. Thank bro
> 
> yes don't use default network node which is _something_network (I forgot its name). Define node URL manually, either by installing a node yourself or find a public node.

thank bro, I'll try.

**Comment by kdimentionaltree** [comment]
Hello, this issue should be fixed in v2.0.63
